cylc/flow/cfgspec/suite.py

```diff
@@ -295,6 +291,15 @@ def upg(cfg, descr):
     u.obsolete('7.8.1', ['runtime', '__MANY__', 'events', 'reset timer'])
     u.obsolete('8.0.0', ['runtime', '__MANY__', 'job', 'shell'])
 
+    # TODO: not obvious how to do this using upgrader
+    try:
+        dependencies = cfg['scheduling']['dependencies']
+        for key, value in dependencies.copy().items():
+            if isinstance(value, dict) and 'graph' in value:
+                dependencies[key] = value['graph']
+    except KeyError:
+        pass
+
```

Comment:
@hjoliver It is not obvious to me how I can do this with the `upgrader`. Can you advice?

Reply:
I'll take a look, but I fear my old deprecation and upgrade mechanism may have reached its limits!

Reply:
(No problem, as done without the upgrader).